opentimelineio/adapters/cmx_3600.py

```diff
@@ -775,26 +775,28 @@ def get_content_for_track_at_index(self, idx, title):
             prv = track[idx-1] if idx > 0 else None
 
             if isinstance(prv, otio.schema.Transition):
-                event = DissolveEvent(
-                    events[-1] if len(events) else None,
-                    prv,
-                    child,
-                    self._tracks,
-                    track.kind,
-                    self._rate,
-                    self._style
+                events.append(
+                    DissolveEvent(
+                        events[-1] if len(events) else None,
+                        prv,
+                        child,
+                        self._tracks,
+                        track.kind,
+                        self._rate,
+                        self._style
+                    )
                 )
-            else:
-                event = Event(
-                    child,
-                    self._tracks,
-                    track.kind,
-                    self._rate,
-                    self._style
+            elif not isinstance(child, otio.schema.Gap):
```

Comment:
I follow what the code is doing, but can you write a comment explaining this if statement?  Is it just that in EDL gaps are expressed as missing record timeline?